### PR TITLE
#159431016 Integrate heroku 

### PR DESCRIPTION
**Pull request by petermwash**
#### What does this PR do?
Seeks to merge the Heroku integration to develop branch.

#### Description of tasks to be completed
Add continuous integration with Heroku.

#### How should this be manually tested?

>Open your postman and send requests to various endpoints as described on the README.
>You should add the host as "ah-titans-api.herokuapp.com", without quotes, before the endpoints.
>You should be able to see results as shown on the screenshots below.

screen shot 2018-08-02 at 17 30 51

screen shot 2018-08-02 at 17 29 47

 ###Any background context you want to provide?
Continuous deployment is enabled for the develop branch.

#### What are the relevant Pivotal Tracker Stories?
[#159431016](https://www.pivotaltracker.com/story/show/159431016)

### CODE REVIEW

**Comment by janetnim** [comment]
I have tested the app on Heroku and it works. Nice

**Review by johnmusiu — APPROVED** [review]
Tested and it works.

**Comment by ronaldndirangu** [comment]
I have tested it and its working as expected. Nice job.

File: authors/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost', 'ah-titans-api.herokuapp.com']
```
Comment (ridgekimani) [inline authors/settings.py:27]: Use .herokuapp.com because of the continuous integration

**Pull request merged**